### PR TITLE
Pr/2395 updated

### PR DESCRIPTION
**Pull request by jquense**
updated #2395

### CODE REVIEW

**Comment by taion** [comment]
@jquense Should `<Modal.Header>` use a `.sr-only` label as well, instead of `aria-label`?

**Comment by jquense** [comment]
I didn't change it because the upstream docs use `aria-label` in that case. There may be more support for it perhaps? Or you just mean the close button in Modal.Header?

**Comment by taion** [comment]
the upstream docs use an `aria-label` for the alert close button as well: http://getbootstrap.com/javascript/#alerts-usage

i thought there was something with the `.sr-only` label working better with more SRs, though

**Comment by jquense** [comment]
The usage is slightly different tho, since in this case one is a Button and the other is jsut a div. I think the problem is due to JAWS handling of button text, not just it's reading of aria-label (which is does generally)

**Comment by taion** [comment]
what do you mean? it's a `<button>` in both cases, no?

**Comment by jquense** [comment]
what aria-label are we talking about? I was looking at the Modal.Title, do you jsut mean the close button there?

**Comment by jquense** [comment]
wait...

**Comment by jquense** [comment]
wtf

**Comment by jquense** [comment]
nvm I'm confusing two things. yes we should make this consistent with the alert close btn

File: src/ModalHeader.js [hunk 1]
```diff
@@ -65,9 +65,9 @@ class ModalHeader extends React.Component {
           <button
             type="button"
             className="close"
-            aria-label={label}
             onClick={createChainedFunction(modal && modal.onHide, onHide)}
           >
+            <span className="sr-only">{closeLabel}</span>
```
Comment (jquense) [inline src/ModalHeader.js]: order shouldn't matter but i can change it. I do want to fix the naming, i forgot this was going into `next` 👍

Comment (taion) [inline src/ModalHeader.js]: better to be consistent than inconsistent between two parallel use cases :p

Comment (jquense) [inline src/ModalHeader.js]: better yet just make it a component

**Pull request merged**